odoo/addons/base/models/ir_sequence.py

```diff
@@ -196,6 +197,19 @@ def write(self, values):
         self.flush_model(values.keys())
         return res
 
+    @api.depends('prefix', 'suffix', 'padding', 'use_date_range', 'date_range_ids', 'date_range_ids.date_from', 'date_range_ids.date_to', 'date_range_ids.number_next_actual', 'number_next_actual')
```

Comment:
why does it depends on `padding` ?

Reply:
We will have to update the preview based on changes in the `padding` field and for that reason, it is added in depends (was added while validation).

Reply:
but we don't use the padding in the compute, why do we want to do that ?

Reply:
the `get_next_char` uses the padding, thus change in the padding field should be reflected in the preview

Reply:
ah ok, thanks for explanation :)